### PR TITLE
Update android-samsung-knox-mobile-enroll.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed KME to a supported scenario.

Reference - https://docs.samsungknox.com/admin/knox-manage/use-samsung-knox-mobile-enrollment.htm

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/enrollment/android-samsung-knox-mobile-enroll.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Still need help? Check out the complete [KME User Guide](https://docs.samsungkno
 
 - **Device Owner support:** - **Device Owner support:** Intune supports enrolling Dedicated and Fully Managed devices by using the KME portal. Other Android enterprise device owner modes will be supported as they become available in Intune.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/android-samsung-knox-mobile-enroll.md:135]:
```suggestion
- **Device Owner support:** Intune supports enrolling Dedicated and Fully Managed devices by using the KME portal. Other Android enterprise device owner modes will be supported as they become available in Intune.
```

File: memdocs/intune/enrollment/android-samsung-knox-mobile-enroll.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ Still need help? Check out the complete [KME User Guide](https://docs.samsungkno
...
 
-- **No work profile support:** KME is a corporate device enrollment method and devices enrolled in Android personally-owned work profile ensure work and personal data are separate on personal devices. So, device enrollment to personally-owned work profile using KME isn't a supported scenario in Intune.
+- **work profile support:** KME is a corporate device enrollment method and devices enrolled in Android personally-owned work profile ensure work and personal data are separate on personal devices. Device enrollment to personally-owned work profile using KME is a supported scenario in Intune.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/android-samsung-knox-mobile-enroll.md:137]:
```suggestion
- **Work profile support:** KME is a corporate device enrollment method and devices enrolled in Android personally-owned work profile ensure work and personal data are separate on personal devices. Device enrollment to personally-owned work profile using KME is a supported scenario in Intune.
```

File: memdocs/intune/enrollment/android-samsung-knox-mobile-enroll.md [hunk 3]
```diff
@@ -134,7 +134,7 @@ Still need help? Check out the complete [KME User Guide](https://docs.samsungkno
...
 
 - **Factory reset to enroll to Android enterprise:** If repurposing devices that have already been set up, devices need to be factory reset when enrolling to Android enterprise.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/android-samsung-knox-mobile-enroll.md:139]:
```suggestion
- **Factory reset to enroll to Android enterprise:** If repurposing devices that have already been set up, devices need to be factory reset when enrolling to Android Enterprise.
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by PRMerger5** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please merge this PR. Thanks

**Review by lenewsad — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**